packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -143,6 +143,7 @@ const createDoesRetentionExist = name => {
 
 const getInitialState = () => ({
   $pool: {},
+  advancedSettings: false,
```

Comment:
`displayAdvancedSettings` whould be clearer IMHO.

Reply:
it's done.